docker-compose.test.yml

```diff
@@ -0,0 +1,70 @@
+version: "2"
+services:
+
+  redis:
+    container_name: redis
+    image: redis:6.0.3
+    ports:
+      - 6379:6379
+    volumes:
+      - redis_data_test:/data
+
+  memcached:
+    container_name: memcached
+    image: memcached:1.6.6
+    ports:
+      - 11211:11211
+
+  pg:
+    container_name: pg
+    image: postgis/postgis:12-3.0
+    environment:
+      POSTGRES_USER: 'inaturalist'
+      POSTGRES_PASSWORD: 'inaturalist'
+      POSTGRES_DB: inaturalist_test
+    ports:
+      - 5432:5432
+    volumes:
+      - pg_data_test:/var/lib/postgresql/data
+      - ./schema/database.sql:/docker-entrypoint-initdb.d/database.sql
+
+  es:
+    container_name: es
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.9.1
+    environment:
+      - discovery.type=single-node
+      - xpack.security.enabled=false
+    ports:
+      - 9200:9200
+    volumes:
+      - es_data_test:/usr/share/elasticsearch/data
+    command: >
+      /bin/sh -c "bin/elasticsearch-plugin list | grep -q analysis-kuromoji
+      || bin/elasticsearch-plugin install analysis-kuromoji;
+      /usr/local/bin/docker-entrypoint.sh"
+
+  api-test:
+    container_name: api-test
```

Comment:
I needed to add `platform: linux/amd64` to get this to work on my ARM-based Mac because node-mapnik will not install otherwise. Any harm in adding that in for all users?

Reply:
Forced the platform to amd64 in the Dockerfile